A_Getting_started/intro.rst

```diff
@@ -32,7 +32,7 @@ user, you are likely to find yourself in one of the following scenarios.
    you’re modeling an animal or making a new detailed joint model. Such
    bottom-up modeling is described in the tutorial "`Getting Started:
    AnyScript
-   Programming <../A_Getting_started_anyscript/intro.docx>`__".
+   Programming <../A_Getting_started_anyscript/intro.html>`__".
```

Comment:
Thanks for the hard work. Be aware of these direct links to html files. They will not work if we at some point use sphinx to generate a PDF version of the tutorials. 

So this should be changed to 
``` 
:doc:`Getting Started AnyScript Programming </A_Getting_started_anyscript/intro>`
```